Lib/urllib/parse.py

```diff
@@ -60,6 +72,27 @@
                'https', 'shttp', 'rtsp', 'rtspu', 'sip', 'sips',
                'mms', 'sftp', 'tel']
 
+
+def _scheme_classes(scheme, overrides=SchemeFlag(0)):
+    """Find out what scheme classes a given scheme fits in.
+
+    This consults the variables uses_relative, uses_netloc, and
+    uses_params. It returns a set of all the classes that apply, with
+    at least the unique classes specified by the optional overrides
+    parameter.
+    """
+    if scheme in uses_relative:
+        overrides |= SchemeFlag.RELATIVE
+
+    if scheme in uses_netloc:
+        overrides |= SchemeFlag.NETLOC
+
+    if scheme in uses_params:
+        overrides |= SchemeFlag.PARAMS
+
```

Comment:
Remove the three `SchemeFlag` references above (again, global level access to the enum members).